doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -937,18 +937,26 @@ This will clean up the mess that can occur when debug logs are enabled.
 To launch your Java application:
 
 <ol>
-	<li>
-		Save the application code into FirstKaaDemo.java file located in the **demo_app** directory.
-	</li>
-	<li>
-		Build the application by running the following command from the **demo_app** directory.
-		
-		<pre>
-			javac -cp *.jar *.java
-		</pre>
-		
-	</li>
-	<li>
+    <li>
+        Save the application code into FirstKaaDemo.java file located in the **demo_app** directory.
+    </li>
+    <li>
+        Download the slf4j-simple-1.7.21.jar in the **demo_app** directory.
+
+        <pre>
+             wget http://central.maven.org/maven2/org/slf4j/slf4j-simple/1.7.21/slf4j-simple-1.7.21.jar
```

Comment:
There is a Windows-specific launch instruction on the next step. Since there is no wget on Windows, maybe we should give specific instructions in this point?